packages/components/calendar/src/instance.ts

```diff
@@ -0,0 +1,3 @@
+import type DateTable from './date-table.vue'
+
+export type DateTableInstance = InstanceType<typeof DateTable>
```

Comment:
Should we prefix this type with component name in the future? like: `CalendarDateTableInstance`.

Reply:
Good suggestion, we can do that right now by export an alias for `DateTableInstance`, WDYT?

Reply:
I agree, this will not cause breaking changes.